.github/workflows/node.js.yaml

```diff
@@ -0,0 +1,24 @@
+name: Node.js CI
+
+on: [push, pull_request]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x, 14.x, 15.x]
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v2
+        with:
+          node-version: ${{ matrix.node-version }}
+      - run: npm ci
+        working-directory: server
+      - run: npm run build --if-present
```

Comment:
@pcgamer1 where is the npm run build command specified in the package.json server?  I saw the logs for this job on the Checks tab and I see that npm run build doesn't run, I think because of the if-present parameter it doesn't even fail, that's why the job is a success.

Ideally we would want the following commands:
npm install (install dependencies)
npm run test 

for now building and deploying isn't happening anywhere, so as long as the tests run it's cool

Reply:
We will add the cmd for running tests in the swagger2 -> oas3 PR, as there are no tests present currently. Removed the cmd for build. Added cmd for eslint check.